app/views/journals/_journal.html.erb

```diff
@@ -0,0 +1,11 @@
+<tr >
+  <td>1 Lesson <br/>
```

Comment:
this data should be uniq for each lesson

Reply:
I don't know how to do this.